timewarp_gl/plugin.cpp

```diff
@@ -428,29 +602,68 @@ class timewarp_gl : public threadloop {
         assert(distortion_indices_data != nullptr && "Timewarp allocation should not fail");
         glBufferData(GL_ELEMENT_ARRAY_BUFFER, num_distortion_indices * sizeof(GLuint), distortion_indices_data, GL_STATIC_DRAW);
 
-        if (enable_offload) {
```

Comment:
Do we want to keep the function of `offload_data` to save rendered textures? That plugin is not currently used and is not supported by `timewarp_vk`, but in the long run it may be of use (e.g. for offloading rendering and debugging).

Reply:
I've re-enabled it for now, but had to make some changes to `readTextureImage()` in order for the entire texture to be copied appropriately.